ledger/accountdb.go

```diff
@@ -1735,8 +1734,11 @@ func performResourceTableMigration(ctx context.Context, tx *sql.Tx, log func(pro
 			return err
 		}
 		insertResourceCallback := func(ctx context.Context, rowID int64, cidx basics.CreatableIndex, ctype basics.CreatableType, rd *resourcesData) error {
-			encodedData := protocol.Encode(rd)
-			_, err = insertResources.ExecContext(ctx, rowID, cidx, ctype, encodedData)
+			var err error
```

Comment:
this handles a case when an account does not have any resources. The callback is called anyway to let correct accounting in other use cases.